doc/content/getting-started/migrating-from-v2/export-v2-devices.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Export End Devices From V2
+weight: 40
+---
+
+In this step, the end devices from {{% ttnv2 %}} will be exported in a JSON format that can then be parsed and imported by {{% tts %}}.
+
+For the commands below, an application on {{% ttnv2 %}} is assumed, with **AppID** `v2-application` and **AppEUI** `0011223300112233`.
+
+Select that **AppID** and **AppEUI** with:
+
+```bash
+$ ttnctl applications select
+```
+
+After selecting the application, make sure that you can list the available end devices:
+
+```bash
+$ ttnctl devices list
+```
+
+In order to export a single device, use the following command. The device will be saved to `device.json`.
+
+```bash
+$ ttnctl devices export "DeviceID" --frequency-plan-id EU_863_870 > device.json
```

Comment:
We don't support capitals in device IDs, so `DeviceID` is a weird example

Reply:
changed to `device_id` (and `app_id`)